bylaws.tex

```diff
@@ -0,0 +1,288 @@
+
+\begin{enumerate}
+
+\item Annual financial statements.
+
+\item The amendment or restatement of these bylaws or the articles of organization.
+
+\item The merger, consolidation, or dissolution of the corporation.
+
+\item The sale, lease, exchange, pledge, or mortgage of any substantial part of the assets of the Corporation, or any contract, option, or other agreement with respect thereto.
+
+\item Such other matters as to which the Board of Directors determines by resolution that such submission is advisable.
+
+\end{enumerate}
+
+\section{Number, Election, and Tenure}
+
+At the regular annual meeting, the existing Directors shall elect Directors for terms commencing on the first day of the next September and continuing for three years and until their respective successors are elected and take office. Persons who are Directors by virtue of election are referred to as `elected Directors' in these Bylaws. The number of elected Directors shall not be less than four or more than ten. In addition, unless already Directors by virtue of election, the President, the Treasurer, and the House Manager of the Eta Tau Chapter of Beta Theta Pi Fraternity shall be ex officio Directors of the Corporation during the times such persons occupy their offices as President, Treasurer, and House Manager, respectively, of the Eta Tau Chapter of Beta Theta Pi Fraternity. These three Directors are referred to as `ex officio Directors' in these Bylaws.
+
+In order to be elected and in order to hold office, each elected Director must be a person who was conferred a Bachelor's Degree or Master's Degree by Worcester Polytechnic Institute, and at the time so conferred was an initiated Brother in good standing of the Eta Tau Chapter of Beta Theta Pi Fraternity.
+
+In the event that the number of elected Directors falls to less than four, through resignations or any other cause, the Board of Directors shall take no action other than the election of Directors until the number of elected Directors is at least four.
+
+\section{Staggered Terms}
+
+The elected Directors shall be divided into three groups about equal in number with the Directors comprising each group having terms of office that expire in a different year than of Directors in the other groups, so that the terms of office of about one-third of elected Directors expire each year.
+
+\section{Regular Meetings}
+
+A regular annual meeting of the Board of Directors shall be held in September of each year, at such time and place as shall be designated by the President or Vice President in the notice of the meeting for the purpose of electing Directors and officers and for the transaction of such other business as may come before the meeting. The Board of Directors may provide by resolution for the holding of additional regular meetings.
```

Comment:
I think we should change this to be August, so that the meeting can happen prior to the academic year starting

Reply:
ok